x11-misc/qt5ct/qt5ct-0.22.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qmake-utils
+
+DESCRIPTION="This program allows users to configure Qt5 settings"
+HOMEPAGE="http://qt5ct.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+RDEPEND=">=dev-qt/qtcore-5.4.0
+	>=dev-qt/qtgui-5.4.0
+	>=dev-qt/qtwidgets-5.4.0"
+DEPEND="${RDEPEND}
+	>=dev-qt/linguist-tools-5.4.0"
```

Comment:
QT deps could be handled by `add_qt_dep()` from `kde5-functions.eclass` and setting `QT_MINIMAL=5.4`